src/components/header/gist-modal/renderer.tsx

```diff
@@ -166,7 +166,7 @@ class GistModal extends React.PureComponent<Props, State> {
         });
         return res.json();
       })
-      .then(json => {
+      .then(async json => {
```

Comment:
Use `await` instead of `then`

Reply:
I've refactored the whole function to use async/await.
The code diff is less/displayed better in VS Code

Reply:
thanks!